tests/test_collections.py

```diff
@@ -128,3 +138,446 @@ def test_add_to_collection(
+            )
+            accepted_page.scroll_into_view(accepted_page.submit_button.element)
+            accepted_page.submit_button.click()
+            accepted_page.loading_indicator.here_then_gone()
+
+            # Navigate to the Removed Page
+            removed_page = CollectionModerationRemovedPage(
+                driver, provider=collection_provider
+            )
+            removed_page.goto()
+            assert CollectionModerationRemovedPage(driver, verify=True)
+            removed_page.loading_indicator.here_then_gone()
+
+            # Find the card for the project that was just removed and click the link for
+            # this project to go to the Project Overview Page. It should be the first one
+            # in the table since the default sort order is by Date (newest first).
+            removed_card = removed_page.get_submission_card(collection_project.id)
+            assert (
+                removed_card.find_element_by_css_selector(
+                    '[data-test-review-action-comment]'
+                ).text
+                == '— Removing collection submission via selenium automated test.'
+            )
+            removed_card.find_element_by_css_selector(
+                '[data-test-submission-card-title]'
+            ).click()
+
+            # On the Project Overview Page, verify that the Collection Container is absent
+            # since the project is no longer included in the collection.
+            project_page = ProjectPage(driver, verify=True)
+            assert project_page.collections_container.absent()
+
+            # Logout and then login as User Two
+            logout(driver)
+            safe_login(
+                driver, user=settings.USER_TWO, password=settings.USER_TWO_PASSWORD
+            )
+
+            # Navigate to the Project Overview page for the project that was just removed
+            # from the collection and verify that the user that created and submitted the
+            # project can see the reason that it was removed.
+            project_page = ProjectPage(driver, guid=collection_project.id)
+            project_page.goto()
+            assert project_page.collections_container.present()
+            project_page.collections_container.click()
+            assert (
+                'Removed from Selenium Post Moderation'
+                in project_page.first_collection_label.text
+            )
+            project_page.collection_justification_link.click()
+            assert (
+                project_page.collection_justification_reason.text
+                == 'Removing collection submission via selenium automated test.'
+            )
+        finally:
+            # Since we are switching logins to User Two we must make sure that we are
+            # properly logged out before proceeding to the next test case.
+            logout(driver)
+
+    def test_post_moderation_collection_remove_by_project_admin(
```

Comment:
As discussed, add a check from the moderator to confirm project removed. Need a `try ... finally ...` due to the update.